client/blocks/importer/wordpress/upgrade-plan/constants.tsx

```diff
@@ -49,23 +50,46 @@ export const UpgradePlanHostingTestimonials = [
 	},
 ];
 
-export const defaultHostingDetails = [
-	{
-		id: 'reduced-error-rate',
-		title: translate( 'Reduced error rate' ),
-		description: translate( '16% fewer errors' ),
-		icon: shield,
-	},
-	{
-		id: 'increased-speed',
-		title: translate( 'Increased speed' ),
-		description: translate( '30% faster' ),
+// TODO: Move this since it has calcs and doesn't fit very well in constants.
+const higherSpeedPercentage = Math.round(
+	( cwvtechReportJson[ 'WordPress.com' ].goodLCP - cwvtechReportJson[ 'WordPress' ].goodLCP ) * 100
+);
+
+const fasterResponsePercentage = Math.round( cwvtechReportJson[ 'WordPress.com' ].goodFID * 100 );
+
+const higherAvailabilityPercentage = 3;
+
+export const defaultHostingDetails: {
+	[ key: string ]: {
+		title: string;
+		description: string | ReactNode;
+		icon: JSX.Element;
+	};
+} = {
+	'higher-speed': {
+		title: translate( 'Higher speed' ),
+		description:
+			translate( '%(higherSpeedPercentage)d 30% faster.', {
+				args: { higherSpeedPercentage },
+			} ) + '*',
```

Comment:
The '*' here and the following one are not part of the translation string because it's more a visual element than the text itself, and they can be changed to something else depending on the page presentation.